sonar-plugin-api/src/test/java/org/sonar/api/utils/text/JsonWriterTest.java

```diff
@@ -39,6 +39,8 @@
 
 public class JsonWriterTest {
 
+  private static final String EMPTY_STRING = "";
```

Comment:
Minor - you can use StringUtils.EMPTY from commons-lang. Having such dependencies is accepted in tests.